.azure/pipelines/ci.yml

```diff
@@ -72,6 +78,12 @@ variables:
     value: -ExcludeCIBinaryLog
   - name: WindowsArm64InstallersLogArgs
     value: -ExcludeCIBinaryLog
+  # Variables for source indexing afterBuild step and job.
+  - name: sourceIndexPackageVersion
+    value: 1.0.1-20210614.1
+  - name: sourceIndexPackageSource
+    value: https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
```

Comment:
@alexperovich as a first step toward breaking eng/common/templates/job/source-index-stage1.yml into two step templates or a step and job template, could these two variables be added to the group below❔ That would help to keep things in sync.

Reply:
These aren't secrets, so it doesn't really make sense to stick them in the key vault. These should generally be flowed by arcade so the repos get updates when I make a new version.

Reply:
I will probably make these parameters to the step templates with defaults.

Reply:
Thanks @alexperovich. Making these parameters to the step templates makes sense. I was mostly thinking the versions might change before we could switch to use those templates. Wanted to cede ownership of the versions in the meantime 😄